redisinsight/ui/src/components/database-overview/DatabaseOverview.tsx

```diff
@@ -1,47 +1,146 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import cx from 'classnames'
 import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiToolTip } from '@elastic/eui'
+import MoreInfoPopover from 'uiSrc/components/database-overview/components/MoreInfoPopover'
+import { RedisModuleDto } from 'apiSrc/modules/instances/dto/database-instance.dto'
+import { getResolutionLimits } from './utils/resolutionHelper'
+import { IMetric } from './components/OverviewMetrics/OverviewMetrics'
```

Comment:
please add index.ts file to OverviewMetrics folder to reduce import string ./components/OverviewMetrics